helm_deploy/hmpps-template-typescript/values.yaml

```diff
@@ -2,6 +2,9 @@ generic-service:
   nameOverride: hmpps-template-typescript
   productId: "UNASSIGNED" # productId for the product that this belongs too, i.e. DPS001, see README.md for details
 
+  # the IRSA service account name for accessing AWS resources
+  # serviceAccountName: "UNASSIGNED"
```

Comment:
if we called this hmpps-template-typescript then it would be renamed to the same name as the project which might be a sensible default

Reply:
Updated 👍